slack-plugin/src/main/kotlin/slack/gradle/bazel/JvmProjectSpec.kt

```diff
@@ -0,0 +1,184 @@
+package slack.gradle.bazel
+
+import com.android.build.gradle.internal.publishing.AndroidArtifacts
+import okio.FileSystem
+import okio.Path
+import org.gradle.api.DefaultTask
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.component.ModuleComponentIdentifier
+import org.gradle.api.artifacts.result.ResolvedArtifactResult
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.provider.SetProperty
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.UntrackedTask
+import org.gradle.internal.component.external.model.ModuleComponentArtifactIdentifier
+import slack.gradle.register
+
+// TODO what about java files? What does the kt_jvm_library rule do?
```

Comment:
srcs supports both Kotlin and Java files

https://github.com/bazelbuild/rules_kotlin/blob/master/docs/kotlin.md#kt_jvm_library-srcs